### ISSUE TITLE
Automatic packaging workflow does not work properly

### ISSUE THREAD

**Issue by FangSen9000** [pnpm/pnpm#5201]
I run a workflow on GitHub that automatically packages and creates RPM / docker image packages. As a result, they display pnpm: not found
The rpm and the generation of docker images cannot be carried out because of the lack of pnpm.  Why did yarn work before, perhaps because the workflow comes with yarn?

### pnpm version:
6.32.11
### Code to reproduce the issue:
![image](https://user-images.githubusercontent.com/72308243/184355194-dc69d2f6-0ff9-4370-a3a3-6ae4883eab13.png)

##makefile
![image](https://user-images.githubusercontent.com/72308243/184355331-0530b366-ef67-4faf-b7fd-74719e773fcc.png)

### Expected behavior:

The workflow for automatically packaging and creating RPM / docker image packages should work normally

### Actual behavior:

![image](https://user-images.githubusercontent.com/72308243/184355468-a9ea6983-6c5d-4399-9d0c-e349f38410f8.png)

![image](https://user-images.githubusercontent.com/72308243/184354263-5473c97a-73b5-42e8-b1ba-c652a9113200.png)

### Additional information:

My work branch：
https://github.com/apache/apisix-dashboard/runs/7805563763?check_suite_focus=true

 
12x， 14x， 16x
 Github

**Comment by FangSen9000** [comment]
To put it simply, there are two GitHub workflows, one is to package and generate rpm, and the other is to package and generate docker image.

It turned out that I used yarn. When i migrated to pnpm, I saw that neither the dockerfile nor the makefile mentioned yarn. I didn't move these two files. Now there is a problem with these two files, saying that pnpm can't be found

How can I install a pnpm when generating docker, or install a pnpm when packaging RPM? How can I use Yum? apt？
